R/modules.R

```diff
@@ -439,6 +439,9 @@ module_labels <- function(modules) {
 #' @keywords internal
 modules_bookmarkable <- function(modules) {
   checkmate::assert_multi_class(modules, c("teal_modules", "teal_module"))
+  if (inherits(modules$children[[1]], "shiny.tag")) {
+    modules$children[[1]] <- NULL
+  }
```

Comment:
`modules_bookmarkable` is used directly only on the `server` side. It is `ui_bookmark_panel` which should receive `modules` without a `shiny.tag` 



```suggestion
```

Reply:
I don't get this. The `ui_bookmark_panel` also calls the `need_bookmarking` which calls `modules_bookmarkable`. Which was causing this issue. Perhaps the solution would be to pass the data module as an attribute instead of `modules$children`.

Reply:
What I meant was that `modules` should be passed to `ui_filter_manager` without `shiny.tag` element.